### DISCUSSION TITLE
Security vulnerability

### DISCUSSION THREAD

**Discussion by ec-hammed** [argoproj/argo-cd#8415] [Q&A] [answered]
https://apiiro.com/blog/malicious-kubernetes-helm-charts-can-be-used-to-steal-sensitive-information-from-argo-cd-deployments/

**Comment by crenshaw-dev** [comment, marked as answer]
Apiiro worked with the Argo CD team, and patches were [released](https://github.com/argoproj/argo-cd/security/advisories/GHSA-63qx-x74g-jcr7) at the same time as the blog post. The Argo CD team has since released a [blog post](https://blog.argoproj.io/argo-cd-deals-with-our-first-zero-day-cve-86e8fb158e8f) about the experience.
